libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -160,7 +160,7 @@ const uint32_t AP_GPS::_baudrates[] = {4800U, 38400U, 115200U, 57600U, 9600U, 23
 // initialisation blobs to send to the GPS to try to get it into the
 // right mode
 const char AP_GPS::_initialisation_blob[] = UBLOX_SET_BINARY MTK_SET_BINARY SIRF_SET_BINARY;
-const char AP_GPS::_initialisation_raw_blob[] = UBLOX_SET_BINARY_RAW_BAUD MTK_SET_BINARY SIRF_SET_BINARY;
+const char AP_GPS::_initialisation_raw_blob[] = UBLOX_SET_BINARY MTK_SET_BINARY SIRF_SET_BINARY;
```

Comment:
At this point we can remove the entire _initialisation_raw_blob as it isn't buying us anything anymore. It was only needed when we were configuring u-blox's to be at different rates depending on if they wanted raw logging.